backend/canvas_app_explorer/canvas_lti_manager/manager.py

```diff
@@ -1,17 +1,38 @@
+import json
 from typing import List, TypedDict
 
 from canvasapi import Canvas
+from canvasapi.exceptions import (
+    BadRequest, CanvasException, Conflict, Forbidden, InvalidAccessToken, RateLimitExceeded,
+    ResourceDoesNotExist, Unauthorized, UnprocessableEntity
+)
 from canvasapi.tab import Tab
 
 from .data_class import ExternalToolTab
+from .exception import CanvasHTTPError
+
+
+EXCEPTION_STATUS_MAP = {
+    BadRequest: 400,
```

Comment:
can we use value as coded from the [HTTPSTATUS library](https://docs.w3cub.com/python~3.9/library/http#http.HTTPStatus)? e.g. `BadRequest: HTTPStatus.BAD_REQUEST.value`

Reply:
This you mean? https://docs.python.org/3.8/library/http.html#http-status-codes That's a good idea, thanks

Reply:
LGTM! Thx!

Reply:
See https://github.com/tl-its-umich-edu/canvas-app-explorer/pull/214/commits/958155f7b8effcdcd383123044f30f0d79974092.

Reply:
Looks good!